src/imitation/algorithms/preference_comparisons.py

```diff
@@ -902,7 +907,7 @@ def _reward_sums(self, fragment_pairs) -> Tuple[np.ndarray, np.ndarray]:
         return np.array(rews1, dtype=np.float32), np.array(rews2, dtype=np.float32)
 
 
-class PreferenceDataset(th.utils.data.Dataset):
+class PreferenceDataset(data_th.Dataset):
```

Comment:
Is this the google style guide?

Reply:
I'm not sure. It's not a very important change anyway, I think, so can just revert it

Reply:
We were already importing this as `data_th` so it seems good to do this for consistency IMO.

Reply:
Cool.